storehaus-mongodb/src/main/scala/com/twitter/storehaus/mongodb/MongoStore.scala

```diff
@@ -93,24 +96,24 @@ class MongoStore[K: MongoValue, V: MongoValue: Manifest] (
 
   protected def getValue(valueObject: MongoDBObject): Option[V] = {
```

Comment:
seems like this method should really be returning `Try[V]` rather than an option and throwing. :/